src/run_spineopt_basic.jl

```diff
@@ -639,25 +639,15 @@ end
 """
 Generalized function to initialize and populate economic parameter values.
 """
-function _initialize_and_populate_values!(
+function _populate_economic_parameter_values!(
     m::Model,
     param_name::Symbol,
     indices_func,
     value_func,
     entity::Symbol,
-    user_outputs
+    user_outputs,
+    key_type
 )
-    
-    key_type = NamedTuple{(
-        entity, :stochastic_scenario, :t),
-        Tuple{SpineInterface.Object, SpineInterface.Object, SpineInterface.TimeSlice}
-    }
-    
-    if !use_economic_representation(model=m.ext[:spineopt].instance)
-        m.ext[:spineopt].values[param_name] = Dict{key_type,Float64}()
-        return nothing
-    end
-
     m.ext[:spineopt].values[param_name] = 
         Dict{key_type,Float64}(
```

Comment:
Why do you need to specify the key_type of the dictionary? I believe just `Dict(...)` will have the same effect here - remember Julia infers the type from the element when possible and I believe this is one of these cases.

Reply:
Thanks, @manuelma; I agree with your comment. I did it because the [Style Guidelines](https://github.com/spine-tools/SpineOpt.jl/blob/master/CONTRIBUTING.md#style-guidelines) proposes it: _"Never leave things without type qualifications. Use an explicit ::Any."_

Unless I didn't understand the Style Guideline (which is a possibility 😝).

Anyway, I think being explicit about types is better for new developers when reading the code and does not harm performance.

Reply:
Do you really find it easier to read? If that's the case then nevermind ;)

Reply:
Btw it looks like we copy-pasted those guidelines from somewhere else - also the previous point "Don't explicitly parameterize types unless it's necessary" seems to contradict the one you're referring?

Reply:
Okay! Maybe we just need to update the guidelines, and then we can simplify this, as you suggest. What do you think?